docs/patterns/sticky-card/index.md

```diff
@@ -0,0 +1,183 @@
+
+{% example palette="lightest",
+           alt="Sticky card naming",
+           src="./sticky-card-style.svg" %}
+
+### Theme
+
+The required elements in a sticky card are a close button, a title or a headline, a call to action, and a container (the light theme container features a drop shadow). The container consists of a background color and two rounded corners on the left or the right side, depending on the orientation. A thin border is also required even if the sticky card background color is different than the page background color.
+
+{% example palette="lightest",
+        alt="Sticky card light theme",
+        src="./sticky-card-theme-light.svg" %}
+
+{% example palette="darkest",
+        alt="Sticky card dark theme",
+        src="./sticky-card-theme-dark.svg" %}
+
+### Layout
+
+A sticky card features header, body, and footer sections, just like a normal Card. Header, body, and footer sections can only include a limited amount of content to ensure that the card doesn’t become too tall.
+
+### Close button
+
+A close button is required for accessibility if a user wants to dismiss the sticky card from their view if they find it distracting.
+
+### Header
+
+The header section can feature only a limited amount of content like a card title or a small headline. This required section introduces what the content is and shouldn’t be hidden.
+
+### Body
+
+To keep the vertical height short, the body section can only include a small amount of text or a small image thumbnail. If a card title and a headline are enough to explain what the content is, the body section can be hidden.
+
+### Footer
+
+The footer section can include normal links or a call to action. A sticky card footer is always required and shouldn't be hidden.
+
+{% example palette="lightest",
+        alt="Sticky card content sections",
+        src="./sticky-card-layout.svg" %}
+
+## Usage
+
+The sticky card placement is different than the normal card placement, where a normal card lives in a predetermined spot on a page. A sticky card slides into view only when a user reaches a specific scroll position on a page. When it does, it stays fixed to the edge of the page and it floats above content and layouts when a user scrolls. It should only be used to feature secondary content that’s helpful to a user, like a resource or webinar.
+
+### Layout
+
+A sticky card can be placed on the left or the right edge of a page and it has a fixed width of 262px.
+
+{% example palette="medium",
+        alt="Sticky card on right side",
+        src="./sticky-card-layout-right.svg" %}
+
+### Content
+
+A sticky card has limited vertical height, so keep content short and only include essential information. A sticky card can include a card title, a headline, text, and a call to action, but not all of these elements need to be included at the same time. Move any extra content to other parts of the page if possible.
+
+{% example palette="lightest",
```

Comment:
here as well

Reply:
Spun out for its own issue, issue with example shortcode and svgs that and double scaled